ghost/core/core/server/models/user.js

```
@@ -11,6 +11,7 @@ const {pipeline} = require('@tryghost/promise');
 const validatePassword = require('../lib/validate-password');
 const permissions = require('../services/permissions');
 const urlUtils = require('../../shared/url-utils');
+const { checkUserPermissionsForRole } = require('./role-utils');
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)